perfkitbenchmarker/benchmark_spec.py

```diff
@@ -254,12 +271,17 @@ def Provision(self):
         sshable_vm_groups[group_name] = [vm for vm in group_vms
                                          if vm.OS_TYPE != os_types.WINDOWS]
       vm_util.GenerateSSHConfig(sshable_vms, sshable_vm_groups)
+    if self.spark_service:
+      self.spark_service.Create()
 
 
```

Comment:
super crazy nit: Looks like it was this way before, but I believe that the convention is just 1 newline in between class methods